examples/Example.h

```diff
@@ -317,19 +317,7 @@ struct GlWindow : Window
 
     GlWindow(Example* example, uint32_t width, uint32_t height) : Window(tvg::CanvasEngine::Gl, example, width, height)
     {
-        SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 3);
-        SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 0);
-        SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_ES);
-        window = SDL_CreateWindow("ThorVG Example (OpenGLES)", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, width, height, SDL_WINDOW_OPENGL | SDL_WINDOW_HIDDEN | SDL_WINDOW_RESIZABLE);
-
-        if (window == nullptr) {
-            // try create OpenGLES context failed, create OpenGL context instead
-            SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 3);
-            SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 3);
-            SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_CORE);
-            window = SDL_CreateWindow("ThorVG Example (OpenGL)", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, width, height, SDL_WINDOW_OPENGL | SDL_WINDOW_HIDDEN | SDL_WINDOW_RESIZABLE);
-        }
-
+        window = SDL_CreateWindow("ThorVG Example (OpenGL/ES)", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, width, height, SDL_WINDOW_OPENGL | SDL_WINDOW_HIDDEN | SDL_WINDOW_RESIZABLE);
```

Comment:
@RuiwenTang Please let me know if this brings up an issue for you. Thanks.

Reply:
If not specify GL context version, on MacOS, SDL will create a GL 2.0 context.

Reply:
@RuiwenTang Please double-check it again. The problem might be with the profile (OpenGL ES), not the version. In my case, it's 4.1 by default. I believe this should work.

Reply:
`4.1` means target on OpenGL context, highest version in GLES is 3.2. And I think the `SDL_GL_CONTEXT_PROFILE_CORE` only valid when target on OpenGL.

Reply:
@RuiwenTang the main point is it. SDL would use the higher version based on the designated profile. And gl engine should work on it.

Reply:
yes 330 core is the minimum version GLEngine required. It works in higher version